README.md

```diff
@@ -1 +1,28 @@
-# Dash-Seedling
+# 🌱 Dash-Seedling
+
+Template repository for building [dash](https://plotly.com/dash/) apps on [FlowEHR](https://github.com/UCLH-Foundry/FlowEHR).
+
+> **Warning**
+> This repository is a _work in progress_. We're working towards a v0.1.0 release
+
+
+## Deploying
+
+### Locally
+
+1. Configure the local confgiruation file
+```
+cp .env.sample .env
+```
+
+2. Run `make` to build the sample Dash Docker container and serve it locally
```

Comment:
Perhaps mention here thet ARM is currently not supported due to Cosmos emulator

Reply:
Opinionated opinion: I prefer good error messages to documentation (I guess both is best but keeping them in sync is a _pain_)

Reply:
opinionated opinion xD love it - personally I like to find out something isn't going to work before I run it

Reply:
you read documentation?!?!

Reply:
haha, fair play. will add something to the README 👍🏼